pkgs/applications/science/misc/gplates/default.nix

```diff
@@ -12,7 +12,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
   buildInputs = [
-    qt4 qwt6_qt4 libGLU libGL glew gdal cgal proj python2
+    qt4 qwt6_qt4 libGLU libGL glew gdal cgal proj7 python2
```

Comment:
This means GPlates is using PROJ 7, but also using GDAL which in turn uses PROJ 8. Should we use `gdal.override { proj = proj7 }` instead?

Reply:
That does seem like the safer option, if GPlates doesn't work with 8.

Reply:
Actually, I don't think that's a good idea: It creates unnecessary load on Hydra and actually we'd need to override GDAL's dependencies as well: GDAL depends on libspatialite, which depend on PROJ. That's a rabbit hole I don't wanna get into.